huggle/login.cpp

```diff
@@ -554,21 +555,20 @@ void Login::PerformLoginPart2(WikiSite *site)
     this->Statuses[site] = WaitingForToken;
     this->LoginQueries.remove(site);
     query->DecRef();
-    query = new ApiQuery(ActionLogin, site);
+    query = new ApiQuery(ClientLogin, site);
```

Comment:
Are you sure that Bot login can use "ClientLogin"?

Reply:
... no, i can't log me in with botpassword...

Reply:
I actually think that classic login was deprecated for everything except for bot passwords, basically in future it should replace classic login with bot password login only.